lib/modules/platform/azure/index.ts

```diff
@@ -747,7 +747,7 @@ export function massageMarkdown(input: string): string {
       'rename PR to start with "rebase!"'
     )
     .replace(regEx(`\n---\n\n.*?<!-- rebase-check -->.*?\n`), '')
-    .replace(regEx(/<!--renovate-debug:.*?-->/), '');
+    .replace(regEx(/<!--renovate-(debug|config-hash):.*?-->/), '');
```

Comment:
Although `debug` and `config-hash` aren't supposed to coexist, but these might be safer.

```suggestion
    .replace(regEx(/<!--renovate-(debug|config-hash):.*?-->/g), '');
```

Or maybe -

```suggestion
    .replace(regEx(/<!--renovate-debug):.*?-->/g), '')
    .replace(regEx(/<!--renovate-config-hash:.*?-->/g), '');
```

Reply:
Always err on the side of keeping it simple / easier maintainability

Reply:
resolved here https://github.com/renovatebot/renovate/pull/17633#discussion_r964373405